edb/pgsql/compiler/relgen.py

```diff
@@ -1535,6 +1535,108 @@ def process_set_as_setop(
     return new_stmt_set_rvar(ir_set, ctx.rel, ctx=ctx)
 
 
+@_special_case('std::EXCEPT')
```

Comment:
I think all three of these are the same except for the value of `subqry.op`, right? Can we have them all be handled by `process_set_as_setop` and just inspect the call to decide which `op` to use?

Reply:
Probably. I copy-pasted them in the beginning because it was easier to separate debugging process without messing up `UNION`.

Reply:
OK, I've folded it back into one function for all 3 operators.